src/containers/AccountConnection.jsx

```diff
@@ -37,10 +37,10 @@ class AccountConnection extends Component {
   connectAccount (auth) {
     const { folderPath } = auth
     const { connector } = this.props
-    let { account } = this.state
+    let { account, successTimeout } = this.state
```

Comment:
Maybe we can comment to know that will concern only the service and not cozy-collect, what do you think? Since I don't see an initial successTimeout in the AccountConnection state. Or maybe we can use 0 as default which will need other changes.